onert-micro/externals/flatbuffers/flexbuffers.h

```diff
@@ -1595,11 +1595,10 @@ class Builder FLATBUFFERS_FINAL_CLASS
   template <typename T> static Type GetScalarType()
   {
     static_assert(flatbuffers::is_scalar<T>::value, "Unrelated types");
-    return flatbuffers::is_floating_point<T>::value
-             ? FBT_FLOAT
-             : flatbuffers::is_same<T, bool>::value
-                 ? FBT_BOOL
-                 : (flatbuffers::is_unsigned<T>::value ? FBT_UINT : FBT_INT);
+    return flatbuffers::is_floating_point<T>::value ? FBT_FLOAT
```

Comment:
maybe we should exclude `externals`

Reply:
I'll add `.FORMATDENY` file into `onert-micro/externals` to ignore format checking.